src/ios-deploy/ios-deploy.m

```diff
@@ -364,12 +366,16 @@ CFStringRef get_device_full_name(const AMDeviceRef device) {
     model_name = dev.name;
     sdk_name = dev.sdk;
     arch_name = dev.arch;
+    product_version = AMDeviceCopyValue(device, 0, CFSTR("ProductVersion"));
+    build_version = AMDeviceCopyValue(device, 0, CFSTR("BuildVersion"));
 
     NSLogVerbose(@"Hardware Model: %@", model);
     NSLogVerbose(@"Device Name: %@", device_name);
     NSLogVerbose(@"Model Name: %@", model_name);
     NSLogVerbose(@"SDK Name: %@", sdk_name);
     NSLogVerbose(@"Architecture Name: %@", arch_name);
+    NSLogVerbose(@"Product Version: %@", product_version);
```

Comment:
It is a good idea to log this. Could you move this to it's own PR? Then we can track it for the next major release where it's more acceptable to break external tools(and hopefully we'll get them to move to JSON so we don't have to worry about our output meant for humans).

This is probably OK, but I'm a little worried about changing the logging output(even only to verbose) since other tools are parsing stdout/stderr very specifically/brittlely. e.g. https://github.com/flutter/flutter/blob/master/packages/flutter_tools/lib/src/ios/devices.dart

Reply:
Sorry for the slow response. Yes, I can make those changes. It won't be for a few more days.